src/main/java/ui/MoveMethodTableModel.java

```diff
@@ -126,8 +132,40 @@ public int getRowCount() {
 
     @Override
     public void setValueAt(Object value, int virtualRow, int columnIndex) {
-        isSelected[virtualRows.get(virtualRow)] = (Boolean) value;
+        final int rowIndex = virtualRows.get(virtualRow);
+        final boolean select = (Boolean) value;
```

Comment:
Maybe `selected` is more appropriate name?

Reply:
`isSelected` already used, so not it's `isRowSelected`.